x-pack/plugins/fleet/dev_docs/data_model.md

```diff
@@ -37,6 +39,8 @@ All of the code that interacts with this index is currently located in
 [`x-pack/plugins/fleet/server/services/agents/crud.ts`](../server/services/agents/crud.ts) and the schema of these
 documents is maintained by the `FleetServerAgent` TypeScript interface.
 
+* Cleanup model:
```

Comment:
We should add to each index if it is a data stream or index. I think it is indicated in the title but it would make it more obvious.

Reply:
Added a catch-all note for this.